frontend/src/features/admin/access-request/AccessRequestPage.test.tsx

```diff
@@ -67,17 +66,17 @@ const store = mockStore({
   },
 });
 describe('AccessRequestPage functionality', () => {
+  const testRender = () =>
+    render(
+      <Provider store={successStore}>
+        <Router history={history}>
+          <AccessRequestPage />
+        </Router>
+      </Provider>,
+    );
   it('renders RequestAccessPage correctly', () => {
-    const tree = renderer
```

Comment:
renderer.create has a separate setup/teardown/act pattern that we aren't using, rather then implement that pattern and the pattern used by react-testing-library, just switch to using react-testing-library for snapshots consistently across the application to avoid act warnings.